mica/archive/tests/test_aca_dark_cal.py

```diff
@@ -78,3 +78,29 @@ def test_get_dark_cal_props(aca_image, allow_negative):
 def test_get_dark_cal_props_table():
     props = dark_cal.get_dark_cal_props_table('2007:001', '2008:001')
     assert np.allclose(props['eb'], [24.6, 25.89, 51.13, 1.9])
+    assert props.colnames == ['ccd_temp', 'date', 'dec', 'dur', 'eb', 'el', 'id', 'l_l0',
+                              'ra', 'start', 'stop', 'sun_el', 'zodib']
+
+
+@pytest.mark.skipif('not HAS_DARK_ARCHIVE', reason='Test requires dark archive')
+def test_get_dark_cal_props_table_acdc():
+    """Just acdc dark cals, giving a non-masked result
+    """
+    props = dark_cal.get_dark_cal_props_table('2019:150', '2019:160')
```

Comment:
I note I missed that I think these new tests do not work with the current archive until we promote the 2019:158 dark cal into it.  I think I'm OK with that.

Reply:
Yeah that was a bit of a chicken and egg that I could not figure a way around.